compose.py

```diff
@@ -177,13 +177,14 @@ def create_argmap(images):
     log("pulling container:{}".format(full_docker_image))
     p = subprocess.run(['docker', 'pull', full_docker_image])
```

Comment:
Why not actually skip the pull?
```suggestion
    if not skip_pull:
        p = subprocess.run(['docker', 'pull', full_docker_image])
        ...
```

Reply:
I feel like it looks cleaner like this since there's minimal logic block change. Note the gpu_min one will have 2 separate code blocks

Reply:
But then it does pull the image, if there is such an image on remote, wouldn't it overwrite the local image and cause confusion to the user?

Reply:
you're right... changed 👍